content/webapp/pages/whats-on/index.tsx

```diff
@@ -90,17 +90,18 @@ import { cacheTTL, setCacheControl } from '@weco/content/utils/setCacheControl';
 const tabItems = [
   {
     id: 'current-and-coming-up',
-    url: '/whats-on',
+    url: `/${prismicPageIds.whatsOn}`,
     text: 'Everything',
   },
   {
     id: 'today',
-    url: '/whats-on/today',
+    // TODO discuss these; do we want to use this technique?
+    url: `/${prismicPageIds.whatsOn}/today`,
```

Comment:
Opinions @wellcomecollection/digital-experience ?

Reply:
I don't have a strong one. This works for me